docs/platform/concepts/maintenance-window.rst

```diff
@@ -17,3 +17,11 @@ In case of **Apache Kafka®** and **OpenSearch®** the service DNS address resol
 .. Note:: 
 
     While the DNS name remains the same, the IP address it points to, will change during a maintenance break. To know more about static IP addresses, check the :doc:`related documentation <static-ips>`.
+
+
+Maintenance updates
+~~~~~~~~~~~~~~~~~~~
+
+Service quarterly patch updates are always mandatory with 6 months advance notice.  Other updates are initially optional. 
```

Comment:
```suggestion
Security updates, platform updates which affect reliability or stability of the service nodes, as well as quarterly patch updates are always mandatory with advance notice of week(s).  Other updates are initially optional. 
```

Reply:
(I wouldn't give exact numbers, but for PG quarterly the number is in 2-3 weeks range usually)